src/App.css

```diff
@@ -1,39 +1,340 @@
+:root {
+  --pale-grey: #f5f6fa;
+  --azure: #0290ff;
+  --black-two: #121212;
+}
+
 .App {
-  text-align: center;
+  display: flex;
+  width: 100%;
+  height: auto;
+  background-color: var(--pale-grey);
+}
+
+.main-wrapper {
+  height: 100%;
+  width: 80%;
+  margin: 0 auto;
+  background-color: #fafafa;
+}
+
+.header {
+  width: 100%;
+  height: 5.938rem;
+  margin: 2rem auto;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.link {
+  margin-right: 3rem;
+}
+
+.logo-navabr {
+  display: flex;
+  justify-content: center;
+}
+
+.logo {
+  width: auto;
+  height: 2.313rem;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--azure);
+}
+
+.book-link {
+  width: auto;
+  height: 1rem;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: var(--black-two);
+}
+
+.category-link {
+  width: auto;
+  height: 1rem;
+  opacity: 0.5;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: var(--black-two);
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.ico-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  width: 2.813rem;
+  height: 2.813rem;
+  border-radius: 50%;
+  border: 1px solid #e8e8e8;
+  background-color: #fff;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.user-icon {
+  font-weight: normal;
+  font-style: normal;
+  font-size: 24px;
+  line-height: 1;
+  letter-spacing: normal;
+  text-transform: none;
+  display: inline-block;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.primary-color {
+  color: #0290ff;
+}
+
+.books-wrapper {
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
 }
 
-.App-link {
-  color: #61dafb;
+.book-card {
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  border-radius: 4px;
+  border: 1px solid #e8e8e8;
+  padding: 2rem;
+  margin-bottom: 1.62rem;
+}
+
+.book-description {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+}
+
+.btn-group {
+  display: flex;
+  align-items: center;
+  margin-top: 1.313rem;
+}
+
+.book-category,
+.book-title {
+  font-weight: 700;
+  color: #121212;
+}
+
+.book-category {
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.875rem;
+  opacity: 0.5;
+}
+
+.book-author,
+.book-title {
+  font-family: "Roboto Slab", serif;
```

Comment:
- [ ] This CSS property is used more than once, it would be nice to use a reusable class which can be reused on the affected components, kindly do so to stick to [best practices](https://speckyboy.com/good-bad-css-practices/)